bom/pom.xml

```diff
@@ -0,0 +1,801 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.geotools</groupId>
+    <artifactId>geotools</artifactId>
+    <version>34-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>gt-bom</artifactId>
+  <packaging>pom</packaging>
+  <name>GeoTools :: Bill of Materials</name>
+  <description>Provides managed dependency versions for GeoTools modules and their consistent transitive dependencies</description>
+
+  <dependencyManagement>
+    <dependencies>
+      <!-- Import platform dependencies -->
+      <dependency>
+        <groupId>org.geotools</groupId>
+        <artifactId>gt-platform-dependencies</artifactId>
+        <version>${project.version}</version>
```

Comment:
this may need to be fully resolved if I recall correctly to an actual version string, eg. during the release procedure otherwise it may use the `project.version` of the actual project importing the bom